RELEASE-NOTES.md

```diff
@@ -1,3 +1,135 @@
 # Release notes for Reactive Streams
 
-Changes will be listed below after version 1.0.0 is released.
\ No newline at end of file
+---
+
+# Version 1.0.1 released on YYYY-MM-DD
+
+After more than two years since 1.0.0, we are proud to announce the immediate availability of `Reactive Streams version 1.0.1`.
+
+## Highlights:
+
+- Specification
+  + A new [Glossary](https://github.com/reactive-streams/reactive-streams-jvm/blob/v1.0.1-RC1/README.md#glossary) section
+  + Description of the intent behind every single rule
+  + No breaking semantical changes
+- Interfaces
+  + No changes
+  + Improved Javadoc
+- TCK
```

Comment:
@akarnokd @ktoso Could you please take a stab at listing the improvements to the TCK? Thank you!
(It's fine to add it as comments here and I'll incorporate it into the release notes.)

/cc @reactive-streams/contributors